Products/CMFPlone/skins/plone_templates/standard_error_message.py

```diff
@@ -27,6 +27,10 @@
 error_traceback = kwargs.get('error_traceback', None)
 error_value = kwargs.get('error_value', None)
 
+if "text/html" not in context.REQUEST.getHeader('Accept', ''):
+    context.REQUEST.RESPONSE.setHeader("Content-Type", "application/json")
+    return '{"error_type": "%s"}' % error_type
```

Comment:
if we add new code we should use the ``.format`` style string templates, shouldnt we?

``'{"error_type": "{0:s}"}'.format(error_type)``